src/app/activity/bounty/Create.ts

```diff
@@ -222,6 +253,9 @@ export const generateBountyRecord = async (
     if (createRequest.isIOU) {
         status = BountyStatus.complete;
     }
+    
+    const bountyCreationChannel = await DiscordUtils.getTextChannelfromChannelId(createdInChannel);
+    const bountyCreationChannelCategory = await DiscordUtils.getTextChannelfromChannelId(bountyCreationChannel.parentId as string);
```

Comment:
For a thread, parentId will reference the channel name. For a channel, parentId will reference the parent category. How are you thinking about what tags get propagated for bounties created in a thread? Should/do we allow bounty lifecycle events in a thread given threads close within 7 days of inactivity?

Reply:
I removed the ability to create in threads for now. It has other side effects that will require more work.